<ui version="4.0" >
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Person data Administrator</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>28</y>
     <width>598</width>
     <height>544</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Connected to:</string>
        </property>
        <property name="buddy" >
         <cstring>databaseEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="databaseEdit" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Persons</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableWidget" name="personTableWidget" >
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="columnCount" >
          <number>6</number>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="refreshButton" >
           <property name="text" >
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Personal information</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="pictureLabel" >
               <property name="sizePolicy" >
                <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="text" >
                <string>no pic</string>
               </property>
               <property name="buddy" >
                <cstring>fileButton</cstring>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="fileButton" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Name:</string>
             </property>
             <property name="buddy" >
              <cstring>nameEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="nameEdit" />
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Prename:</string>
             </property>
             <property name="buddy" >
              <cstring>preNameEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLineEdit" name="preNameEdit" />
           </item>
           <item row="5" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Tag id:</string>
             </property>
             <property name="buddy" >
              <cstring>tagIdSpin</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QSpinBox" name="tagIdSpin" >
             <property name="maximum" >
              <number>10000</number>
             </property>
             <property name="value" >
              <number>2213</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Description:</string>
             </property>
             <property name="buddy" >
              <cstring>descriptionEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1" >
            <widget class="QTextEdit" name="descriptionEdit" >
             <property name="minimumSize" >
              <size>
               <width>150</width>
               <height>70</height>
              </size>
             </property>
             <property name="lineWidth" >
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Color: </string>
             </property>
             <property name="buddy" >
              <cstring>colorEdit</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="2" >
            <widget class="QPushButton" name="colorButton" >
             <property name="text" >
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <layout class="QHBoxLayout" name="horizontalLayout_3" >
             <item>
              <widget class="QLineEdit" name="colorEdit" >
               <property name="inputMask" >
                <string/>
               </property>
               <property name="maxLength" >
                <number>7</number>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="colorLabel" >
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton" >
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="commitButton" >
           <property name="text" >
            <string>Commit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnect" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>572</y>
     <width>598</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionConnect" >
   <property name="text" >
    <string>&amp;Connect...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="text" >
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDisconnect" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Disconnect ...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>personTableWidget</tabstop>
  <tabstop>refreshButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>preNameEdit</tabstop>
  <tabstop>tagIdSpin</tabstop>
  <tabstop>colorButton</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>commitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
